### PR TITLE
new builtin template and ability to not create operational token

### PR DESCRIPTION
**Pull request by otakup0pe**
Addresses #89 and one I didn't write down ✨

### CODE REVIEW

File: docs/operations.md [hunk 1]
```diff
@@ -12,12 +12,14 @@ There are also some [miscelaneous]({{site.baseurl}}/misc) operations which you m
 
 Every aomi operation can take a `--verbose` flag. By default, the tool is quite silent, using return codes to communicate status. The verbose mode is good for troubleshooting and should not display any sensitive information.
 
-Help for operations should be available with the `--help` argument.
-
 You should be able to trust that meaningful output will be written to stdout. All errors (and verbose information) will be written to stderr.
 
+Help for operations should be available with the `--help` argument.
```
Comment (ericfode) [inline docs/operations.md]: test for --help

File: aomi/cli.py [hunk 2]
```diff
@@ -157,6 +157,11 @@ def base_args(parser):
                         dest='lease',
                         help='Lease time for intermediary token.',
                         default='10s')
+    parser.add_argument('--reuse-token',
+                        dest='reuse_token',
+                        help='Whether to reuse the existing token. Note'
+                        ' this will cause metadata to not be preserved',
```
Comment (ericfode) [inline aomi/cli.py]: Why?

Comment (ericfode) [inline aomi/cli.py]: When would i want this?

File: docs/extract.md [hunk 3]
```diff
@@ -97,3 +97,4 @@ You may list all included builtin templates by invoking `aomi template --builtin
 * `terraform-aws` will render a Terraform AWS `provider` section. Note you will need to pass in the `aws_region` variable as an extra.
 * `json-kv` will render a JSON key-value file.
 * `docker-auth` will render a Docker `config.json` auth snippet. It expects a `user`, `password`, and `url` variable.
+* `shenv` will render a Shell snippet full of environment variables.
```
Comment (ericfode) [inline docs/extract.md]: usage example?

Comment (otakup0pe) [inline docs/extract.md]: There is usage information on builtin templates in the doc page. Nothing detailed yet, will add some better examples in my docs [branch](https://github.com/Autodesk/aomi/compare/master...otakup0pe:freedmj/docs).

File: aomi/cli.py (hunk 2 above)
Comment (otakup0pe) [inline aomi/cli.py]: Justification/context in the docs.

**Pull request merged**